R/converters.R

```diff
@@ -1530,6 +1530,8 @@ MetamorpheusToMSstatsPTMFormat = function(input,
                                                     verbose, 
                                                     log_file_path)
         
+        ptm_input = ptm_input[grepl(mod_id, ptm_input$PeptideSequence),]
```

Comment:
if ptm_input is data.table, the ptm_input$ part is not needed (so is ", " after it)